test/extended/crc/ux/installer/installer_windows.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 const (
-	installerWindowTitle string = "CodeReady Containers Setup"
+	installerWindowTitle string = "CRC Setup"
```

Comment:
Not sure where this string comes from, could not find it by grepping around, have you checked that it's `CRC Setup` and not `OpenShift Local setup`? (I agree CRC is more likely)

Reply:
i think this is for testing the msi, in that case it'd be `Red Hat OpenShift Local setup`